saleor/plugins/base_plugin.py

```diff
@@ -1224,6 +1224,18 @@ def __str__(self):
     # updated.
     voucher_updated: Callable[["Voucher", str, None], None]
 
+    # Trigger when voucher code is created.
+    #
+    # Overwrite this method if you need to trigger specific logic after a voucher code
+    # are created.
```

Comment:
`# is created.`

Reply:
as this is related to multiple codes, so should be "voucher code**s**"